.github/RELEASE_TEMPLATE.md

```diff
@@ -1,150 +1,89 @@
+# Weekly Reports
 
-1. Pull the `docker` image from docker-hub:
+- [Week 12 - 2020-03-20](https://github.com/input-output-hk/cardano-wallet/tree/weekly-reports/2020-03-20)
 
-  ```bash
-  $ docker pull inputoutput/cardano-wallet:YYYY.MM.DD-jormungandr
-  ```
+# Documentation
 
-2. Interact with the image using `docker run`:
+<!-- A snapshot of the documentation at the time of releasing. -->
 
-  ```bash
-  $ docker run inputoutput/cardano-wallet:YYYY.MM.DD-jormungandr --help
-  ```
+| Link                                                                                                                                        | Audience                                                   |
+| ---                                                                                                                                         | ---                                                        |
+| [API Documentation](https://input-output-hk.github.io/cardano-wallet/api/{{GIT_TAG}})                                                       | Users of the Cardano Wallet API                            |
+| CLI Manual: [jormungandr](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-Command-Line-Interface/{{JORM_CLI_WIKI_COMMIT}}) / [byron](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-Command-Line-Interface-(cardano-wallet-byron)/{{BYRON_CLI_WIKI_COMMIT}}) | Users of the Cardano Wallet API                            |
+| [Docker Manual](https://github.com/input-output-hk/cardano-wallet/wiki/Docker/{{DOCKER_WIKI_COMMIT}})                     | Users of the Cardano Wallet API                            |
+| [Haddock Documentation](https://input-output-hk.github.io/cardano-wallet/haddock/{{GIT_TAG}})                                               | Haskell Developers using the `cardano-wallet` as a library |
 
-# Known Issues
+# Installation Instructions 
 
-<!-- Bugs that are known at the moment at the moment of the release but aren't
-show-stoppers. -->
+<!-- Specific installation steps for this particular release. This should
+basically captures whatever is currently available on the repository at
+the moment of releasing. -->
 
-# Key Features
+## Jormungandr / ITN
 
-<!-- What's included in the software, mention what's new in this particular
-release. Use: ![][new] -->
+1. Install [jormungandr@{{JORM_TAG}}](https://github.com/input-output-hk/jormungandr/releases/tag/{{JORM_TAG}}).
 
-## Command-Line
+2. Download the provided `cardano-wallet-jormungandr` for your platform, and uncompress it in a directory that is on your `$PATH`, e.g. `/usr/local/bin`. Or `%PATH%` on Windows.
 
-## Server
+3. (optional) Install the bash/zsh auto-completion script according to the [jormungandr cli manual](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-Command-Line-Interface/{{JORM_CLI_WIKI_COMMIT}})
 
-## Internal
+4. Start `cardano-wallet --help` and see available parameters.
 
-## Known Limitations
+## Byron / Mainnet
```

Comment:
The target component really is `cardano-node` though, not `Byron`. What about the following formulation:

## cardano-node / Byron

Reply:
Done.